.isort.cfg

```diff
@@ -0,0 +1,8 @@
+[settings]
+force_grid_wrap=0
```

Comment:
Most of these are to enforce compatibility with black: https://github.com/timothycrosley/isort/issues/694#issuecomment-564261886